src/flamegraph/attrs.rs

```diff
@@ -113,7 +114,7 @@ pub struct GElementAttrs {
     pub onclick: Option<String>,
 
     /// Extra attributes to include
-    pub extra: HashMap<String, String>,
+    pub extra: IndexMap<String, String>,
```

Comment:
This change is pretty unfortunate, but I see why you made it. Could we maybe instead just make this a `Vec<(String, String)>`? I don't think we actually _need_ it to be a map?

Reply:
You're right, a Vec would do just fine here.

Reply:
You're right, a Vec would do just fine here.